### PR TITLE
Update hi.js

### PR DESCRIPTION
**Pull request by Codewith-Akshut**
Update on Hi.js by akshutcoder

Discord - Thomas Shelby#6969
Ign - akshutcoder

### CODE REVIEW

File: hi.js [hunk 1]
```diff
@@ -82,7 +82,7 @@ settings.import इम्पोर्ट
 settings.export ऐक्सपोर्ट
 settings.none कोई सेटिंग नहीं मिली
 settings.backup बैकअप सेटिंग्स
-settings.controls.change Keybinds
+settings.controls.change कीबइंडस
```
Comment (Inspire-24) [inline hi.js:85]: i think it would be better if left as it is, as it isnt precise

**Review by Inspire-24 — CHANGES_REQUESTED** [review]
Everything other than that looks good!

File: hi.js (hunk 1 above)
Comment (Codewith-Akshut) [reply hi.js:85]: Sure

**Comment by mty22** [comment]
image

**Pull request merged**